### ISSUE TITLE
Install process and documentation thereof is hard to follow

### ISSUE THREAD

**Issue by bericp1** [YXNi/Ni_CSCI2270_Final_Project#1]
IMHO, the documentation in the README and the PDF version of the README need much more detailed installation instructions.

I was able to get the program to open by searching through and finding `VisualC#\serialcom\bin\Debug\serialcom.exe` which seems to be a pre-compiled binary that I got lucky enough to have work in my Windows 8.1 virtual machine.

I would suggest cleaning up the documentation a bit using Github-supported Markdown (see this guide by GitHub [here](https://guides.github.com/features/mastering-markdown/)).

Additionally, I would heavily suggest providing a way for the user to install without having Visual C# installed since (if the pre-compiled binary didn't work) I would have had to have done that.

Just a few pointers/heads up!

**Comment by YXNi** [comment]
Hi Brandon,
   Thanks for the feedback. I was actually working on the documentation and
make the ui better today. I will upload a better version of it. I will
email you when i pull it to github. You can look at the code again if yoy
like. Also you will need an atduino and a motion sensor to make the code.

Yours,
Yiqi

**Comment by YXNi** [comment]
Hi,
   I uploaded better version of this application. It should be better
explained. The reason you wouldn't run the .exe it that you have to change
the serial communication port in the code...

Yiqi
